### PR TITLE
[Mfirewall] Added r2 to list of buckets

### PR DESCRIPTION
**Pull request by marciocloudflare**
Closes PCX-10801

### CODE REVIEW

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md [hunk 1]
```diff
@@ -187,4 +187,39 @@ header: Bucket list response example
...
+## R2
+
+To start collecting packet captures with R2, you first need to configure it properly. For all the required details, refer to the [Cloudflare R2](/r2/) documentation.
+
+### Configure R2
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login), and select you account.
+2. Select **R2** > **Create bucket**.
+3. Give your bucket a name > **Create bucket**.
+4. Go to the R2 Overview page, ans select **Manage R2 API Tokens**.
```
Comment (RebeccaTamachiro) [inline content/magic-firewall/packet-captures/pcaps-bucket-setup.md]:
```suggestion
4. Go to the R2 Overview page, and select **Manage R2 API Tokens**.
```

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md [hunk 2]
```diff
@@ -187,4 +187,39 @@ header: Bucket list response example
...
 
-To learn how to collect packet captures, refer to [Collect packet captures](/magic-firewall/packet-captures/collect-pcaps/).
\ No newline at end of file
+To learn how to collect packet captures, refer to [Collect packet captures](/magic-firewall/packet-captures/collect-pcaps/).
+
+## R2
+
+To start collecting packet captures with R2, you first need to configure it properly. For all the required details, refer to the [Cloudflare R2](/r2/) documentation.
+
+### Configure R2
```
Comment (RebeccaTamachiro) [inline content/magic-firewall/packet-captures/pcaps-bucket-setup.md]:
```suggestion
### Create bucket and API token
```

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md [hunk 3]
```diff
@@ -187,4 +187,39 @@ header: Bucket list response example
...
+curl https://api.cloudflare.com/client/v4/accounts/{account_id}/pcaps/ownership \
+  --header 'Content-Type: application/json' \
+  --header 'X-Auth-Email: <EMAIL>' \
+  --header 'X-Auth-Key: <API_KEY>' \
+  --data '{
+    "destination_conf": "r2://<BUCKET_NAME>?account-id=<ACCOUNT_ID>&access-key-id=<R2_ACCESS_KEY_ID>&secret-access-key=<R2_SECRET_ACCESS_KEY>"
+}'
+```
+
+The [response](/api/operations/magic-pcap-collection-add-buckets-for-full-packet-captures) has a `"filename"` parameter with the name of a file that Cloudflare wrote to your R2 bucket. You need to download it for the next step.
```
Comment (pedrosousa) [inline content/magic-firewall/packet-captures/pcaps-bucket-setup.md]: Nit: Maybe we could show an example response here?

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md (hunk 2 above)
Comment (RebeccaTamachiro) [reply content/magic-firewall/packet-captures/pcaps-bucket-setup.md]: Was just thinking that `configure` here could be confused with the `configure it properly` that you have in the previous paragraph.

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md [hunk 4]
```diff
@@ -187,4 +187,39 @@ header: Bucket list response example
...
+
+## R2
+
+To start collecting packet captures with R2, you first need to configure it properly. For all the required details, refer to the [Cloudflare R2](/r2/) documentation.
+
+### Configure R2
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login), and select you account.
+2. Select **R2** > **Create bucket**.
+3. Give your bucket a name > **Create bucket**.
```
Comment (RebeccaTamachiro) [inline content/magic-firewall/packet-captures/pcaps-bucket-setup.md:200]: For this step, is the option for **Location** (Automatic or Specify jurisdiction) important?

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md [hunk 5]
```diff
@@ -187,4 +187,39 @@ header: Bucket list response example
...
+  --data '{
+    "destination_conf": "r2://<BUCKET_NAME>?account-id=<ACCOUNT_ID>&access-key-id=<R2_ACCESS_KEY_ID>&secret-access-key=<R2_SECRET_ACCESS_KEY>"
+}'
+```
+
+The [response](/api/operations/magic-pcap-collection-add-buckets-for-full-packet-captures) has a `"filename"` parameter with the name of a file that Cloudflare wrote to your R2 bucket. You need to download it for the next step.
+
+### Validate bucket ownership
+
+Refer to the [Validate a bucket](#validate-a-bucket) API instructions, for more details on the entire process to [validate your R2 bucket](/api/operations/magic-pcap-collection-validate-buckets-for-full-packet-captures). When specifying the R2 destination for this validation, exclude the secret and access keys from the URL.
```
Comment (RebeccaTamachiro) [inline content/magic-firewall/packet-captures/pcaps-bucket-setup.md]:
```suggestion
Refer to the [Validate a bucket](#validate-a-bucket) API instructions for more details on the entire process to [validate your R2 bucket](/api/operations/magic-pcap-collection-validate-buckets-for-full-packet-captures). When specifying the R2 destination for this validation, exclude the secret and access keys from the URL.
```

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md (hunk 4 above)
Comment (marciocloudflare) [reply content/magic-firewall/packet-captures/pcaps-bucket-setup.md:200]: It's not. Also, I don't want to over explain something that might change in the future. That's why I called out the "for all the details thing go to R2 docs"

File: content/magic-firewall/packet-captures/pcaps-bucket-setup.md (hunk 3 above)
Comment (marciocloudflare) [reply content/magic-firewall/packet-captures/pcaps-bucket-setup.md]: Added

**Pull request merged**